apps/bblcricket/bbl_cricket.star

```diff
@@ -0,0 +1,771 @@
+                        children = [
+                            render.Box(width = 64, height = 6, child = render.Text(content = "No games", color = "#FFF", font = "CG-pixel-3x5-mono")),
+                        ],
+                    ),
+                    render.Row(
+                        expanded = True,
+                        main_align = "space_between",
+                        cross_align = "end",
+                        children = [
+                            render.Box(width = 64, height = 6, child = render.Text(content = "scheduled", color = "#FFF", font = "CG-pixel-3x5-mono")),
+                        ],
+                    ),
+                ],
+            ),
+        )
+
+def TeamScore(BattingTeam, BattingTeamColor, Wickets, Runs):
+    return render.Column(
+        children = [
+            render.Row(
+                children = [
+                    render.Box(width = 42, height = 8, child = render.Padding(
+                        pad = (2, 1, 0, 0),
+                        child = render.Marquee(
+                            width = 42,
+                            child = render.Text(content = BattingTeam, color = BattingTeamColor, font = "CG-pixel-3x5-mono", offset = 0),
+                        ),
+                    )),
+                    render.Box(width = 22, height = 8, child = render.Text(content = Wickets + "/" + Runs, color = BattingTeamColor, font = "CG-pixel-3x5-mono")),
+                ],
+            ),
+        ],
+    )
+
+def BatsmanScore(Batsman, Runs, BatsmanColor):
+    return render.Row(
+        children = [
+            render.Box(width = 50, height = 8, child = render.Padding(
+                pad = (2, 1, 0, 0),
+                child = render.Marquee(
+                    width = 50,
+                    child = render.Text(content = Batsman, color = BatsmanColor, font = "CG-pixel-3x5-mono", offset = 0),
+                ),
+            )),
+            render.Box(width = 14, height = 8, child = render.Padding(
+                pad = (0, 0, 0, 0),
+                child = render.Marquee(
+                    width = 14,
+                    child = render.Text(content = Runs, color = BatsmanColor, font = "CG-pixel-3x5-mono", offset = 0),
+                ),
+            )),
+        ],
+    )
+
+def StatusRow(StatusMsg, StatusColor):
+    return render.Row(
+        children = [
+            render.Box(width = 64, height = 8, child = render.Text(content = StatusMsg, color = StatusColor, font = "CG-pixel-3x5-mono")),
+        ],
+    )
+
+TeamOptions = [
+    schema.Option(
+        display = "Adelaide Strikers",
+        value = "4843",
+    ),
+    schema.Option(
+        display = "Brisbane Heat",
+        value = "4844",
+    ),
+    schema.Option(
+        display = "Hobart Hurricanes",
+        value = "4845",
+    ),
+    schema.Option(
+        display = "Perth Scorchers",
+        value = "4846",
+    ),
+    schema.Option(
+        display = "Melboure Renegades",
```

Comment:
```suggestion
        display = "Melbourne Renegades",
```

Reply:
I can't believe I spelt that wrong! I must have looked at the dropdown 100s of times during testing 

Great pick up!